### PR TITLE
Option to disable Case Light brightness control

### PR DESCRIPTION
**Pull request by InsanityAutomation**
Add option to prevent brightness control of non-dimmable lights

### CODE REVIEW

**Comment by Roxy-3D** [comment]
That spelling correction needs to be replicated 70x.

**Comment by InsanityAutomation** [comment]
whoops lol ill do that momentarily :)

File: Marlin/src/feature/caselight.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ void update_case_light() {
 
   #else // !CASE_LIGHT_USE_NEOPIXEL
 
-    if (PWM_PIN(CASE_LIGHT_PIN))
+    if (PWM_PIN(CASE_LIGHT_PIN) && DISABLED(CASE_LIGHT_NO_BRIGHTNESS))
```
Comment (thinkyhead) [inline Marlin/src/feature/caselight.cpp]: You can't use `DISABLED` like this. It won't expand. You have to turn it into a run-time-accessible boolean.

Comment (InsanityAutomation) [reply Marlin/src/feature/caselight.cpp]: I blame lack of coffee.... I dont know what I was thinking on that one lol

**Pull request merged**